pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerendervue/structure/page/page.html

```diff
@@ -62,6 +62,8 @@
     </span>
 </div>
 
+<sly data-sly-include="renderer-pre.html"></sly>
```

Comment:
I don't necessarily enjoy this name. Maybe we call this script differently? E.g. `custom-js` or something similar?

Reply:
`renerer-pre.html` indicates that it is executed before `renderer.html` - we could in the future also have a `renderer-post.html` if we choose a different name it would have to support the same idea. I would not enjoy `custom-js` as a name here.

Reply:
Yeah, it's not so important for me, though I'd ask a native speaker to evaluate it, something simply doesn't sound right for me in this name. I mean - maybe `pre-renderer.html` is more correct? I don't know.